src/cli/cli_commissioner.cpp

```diff
@@ -351,6 +544,18 @@ template <> otError Commissioner::Process<Cmd("id")>(Arg aArgs[])
     return error;
 }
 
+/**
+ * @cli commissioner start
+ * @code
+ * commissioner start
+ * Commissioner: petitioning
+ * Done
+ * Commissioner: active
+ * @endcode
+ * @par
+ * Enables the Thread %Commissioner role.
```

Comment:
I think this should be "Start" instead of "Enable", those seem to be two slightly different things in the context of commands.